### DISCUSSION TITLE
EditorOption does not work on DateTimeEditor

### DISCUSSION THREAD

**Discussion by minhhungit** [serenity-is/Serenity#6010] [Help]
Hi team, it looks EditorOption does not work on DateTimeEditor, I checked its properties, there is no `options` property
Latest serenity version

Feel free to move this to discussion if this is not a bug. In the case, how can I add `options` into editor

**Comment by VictorTomaili** [comment]
Do you pass it to super from the constructor? 
This option should work. 
https://github.com/serenity-is/Serenity/blob/master/src/Serenity.Scripts/CoreLib/UI/Editors/DateTimeEditor.ts#L23

If it's not the case, please describe your usage in detail with samples. 
Also, I move this issue to discussions. If we decide it's a bug, then I move it back to the issues.

**Comment by minhhungit** [comment]
hi @VictorTomaili, thank you for replying soon
I don't create editor manual by coding and pass option as parameter into constructor
I define [EditorOption] attribute on Row.cs, for example:

It does not work with DateTimeEditor, maybe other type like Date editor also does not work

```csharp
[DisplayName("Start From"), Column("start_from")]
[DateEditor]
[QuickFilter]
[EditorOption("aaa", "bbb")]
[FilteringOption("xxx", "yyy")]
public DateTime? StartFrom
{
    get => fields.StartFrom[this];
    set => fields.StartFrom[this] = value;
}
```

![image](https://user-images.githubusercontent.com/2279508/136029741-b8adb77b-975c-466e-b766-d68cf95d774d.png)


##**Expecting;**##

![image](https://user-images.githubusercontent.com/2279508/136030148-b55dfb24-f89e-4c5f-8deb-0c571100c401.png)

I'm trying to put order number for quick filters and sort them on UI
